### PR TITLE
feat: update instructions to install findora sdk and minor code clean up

### PR DESCRIPTION
**Pull request by okovalov**

### CODE REVIEW

**Comment by okovalov** [comment]
@mlin2020 @kingpinXD guys, this pr is to add minor updates in the process of setting up the SDK

CleanShot 2021-11-08 at 12 46 12@2x

**Comment by mlin2020** [comment]
Looks good (via quick skim). You basically changed install instruction to reference NPM repo. The rest seems to be minor text visual formatting(?). I did not test the actual install though.

**Comment by okovalov** [comment]
> Looks good (via quick skim). You basically changed install instruction to reference NPM repo. The rest seems to be minor text visual formatting(?). I did not test the actual install though.

yes, that what it is. some formatting and updates in the instructions.  @mlin2020

File: docs/wallet/sdk_integration.md [hunk 1]
```diff
@@ -5,276 +5,365 @@ sidebar_position: 2
...
 Developers should be able to integrate Findora features such as sending FRA to another wallet, checking the FRA balance, etc. into their crypto wallet, crypto exchange or Dapp by following this guide.
 
 ## **Setting Up the `Findora SDK`**
 
-- **1.** **Setup the Findora SDK**
+- **1.** **Install the Findora SDK**
+
+```bash
+npm i @findora-network/findora-sdk.js
+```
```
Comment (Jedi2002) [inline docs/wallet/sdk_integration.md:20]: Hi , I get the following error when I run this 
Is it related to my machine ? 
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /Users/tanmay/.npm/_cacache/tmp/git-clone4Zt1Ea/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/Users/tanmay/.npm/_cacache/tmp/git-clone4Zt1Ea/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tanmay/.npm/_logs/2021-11-10T18_42_50_527Z-debug.log

Comment (okovalov) [reply docs/wallet/sdk_integration.md:20]: @Jedi2002 @ws4charlie guys, please use yarn to install it. please make sure your node is at least version 14

https://user-images.githubusercontent.com/1706749/141207549-e7f943c7-bbce-4f74-83ae-10d2453f5c15.mp4

**Comment by okovalov** [comment]
closing in favor of https://github.com/FindoraNetwork/findora-wiki/commit/5fa859130cfda4e291bb001498fc6eecc2f1b05b

**Pull request closed without merging**